### Post 500
**Post URL**: /t/project-1-llm-based-automation-agent-discussion-thread-tds-jan-2025/164277/500
- **ID**: 595738
- **Author**: Ayush Kumar Shaw  (23f2001286)
- **Created At**: 2025-02-16T04:34:50.606Z
- **Content**:  
  sir I am noticing a pattern, that when I am running the datagen first. And then using the evaluate.py, then I am getting the A2 right.<br>
But running the evaluation.py for the 2nd time cause the A2 to fail…<br>
Probabbly Because the file in the data folder gets upated should I worry for that…
- **Reactions**: None
- **Post Number**: 500

